src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetHttpModule.cs

```diff
@@ -130,6 +130,12 @@ private void OnEndRequest(object sender, EventArgs eventArgs)
 
         private void OnError(object sender, EventArgs eventArgs)
         {
+            if (!Tracer.Instance.Settings.IsIntegrationEnabled(IntegrationName))
```

Comment:
This follows the convention of each callback starting with the `IsIntegrationEnabled` check.

Reply:
Resolving because I've reverted this change.